docs/serving/samples/multi-container/README.md

```diff
@@ -0,0 +1,250 @@
+A simple web app written in Go that you can use for multi container testing.
+
+Follow the steps below to create the sample code and then deploy the app to your
+cluster. You can also download a working copy of the sample, by running the
+following command:
+
+```shell
+git clone -b "{{< branch >}}" https://github.com/knative/docs knative-docs
+```
+
+## Prerequisites
+
+- A Kubernetes cluster with Knative installed and DNS configured. Follow the
+  [installation instructions](../../../install/README.md) if you need to
+  create one.
+- [Docker](https://www.docker.com) installed and running on your local machine,
+  and a Docker Hub account configured (we'll use it for a container registry).
+- Make sure multi-container flag is enabled as part of `config-features` configmap.
+
+## Recreating the sample code
+
+Multi container has more than one container. For testing, we will create two containers: a serving container and a sidecar container.
+The `multi-container` directory already exist with predefined code and dockerfile for serving and sidecar container with service yaml.
+If user wants to update code, dockerfile or service yaml they can follow below steps.
+
+### Serving Container
+1. `cd knative-docs/docs/serving/samples/multi-container/servingcontainer`
+1. Update a file named `servingcontainer.go` and copy the code block below into it. This
+   code creates a basic web server which listens on port 8881:
+
+   ```go
+   package main   
+   import (
+   	"fmt"
+   	"io/ioutil"
+   	"log"
+   	"net/http"
+   )
+   func handler(w http.ResponseWriter, r *http.Request) {
+   	log.Println("serving container received a request.")
+   	res, err := http.Get("http://127.0.0.1:8882")
+   	if err != nil {
+   		log.Fatal(err)
+   	}
+   	resp, err := ioutil.ReadAll(res.Body)
+   	if err != nil {
+   		log.Fatal(err)
+   	}
+   	fmt.Fprintln(w, string(resp))
+   }
+   func main() {
+   	log.Print("serving container started...")
+   	http.HandleFunc("/", handler)
+   	log.Fatal(http.ListenAndServe(":8881", nil))
+   }
+   ```
+1. In your project directory, update a file named `Dockerfile` and copy the code
+   block below into it. For detailed instructions on dockerizing a Go app, see
+   [Deploying Go servers with Docker](https://blog.golang.org/docker).
```

Comment:
```suggestion
1. Copy the following code into the `Dockerfile` file:
```

Reply:
No need to mention directory unless this needs another `cd` command to go somewhere else after the previous steps?
Also, any notes about dockerizing etc should go in an **Additional resources** section or something at the end of the doc, instead of inside a step of a procedure. Steps should only be prescriptive information :slightly_smiling_face: